src/app/components/cards/Comment.scss

```diff
@@ -166,6 +170,7 @@
 .Comment__replies {
   margin-top: 1.4rem;
   margin-left: 62px;
+  border-left: 1px dotted #eee;
```

Comment:
The dotted border looks great on mobile, but on desktop the alignment with avatar is awkward. I think on desktop we need to either remove the border, or add padding between border and avatar

Reply:
@roadscape I did added margin around avatar. The screenshot is not up-to-date.
Here is the new one. 

![Screen Shot 2019-06-19 at 8 36 44 am](https://user-images.githubusercontent.com/310654/59724499-6976e180-926d-11e9-8332-de11d9d200b7.jpg)

Also themiefied the color

Reply:
👍 👍